set-picnic-var.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+export PICNIC_PARAMS_PATH=$PWD/src/sig_picnic/external/Picnic-master/precomputed_data/
+echo $PICNIC_PARAMS_PATH
+
```

Comment:
What's the purpose of this script? There's no explanation in the README of its role.